README.md

```diff
@@ -1,2 +1,95 @@
-# navigation
-Repository for the discussion and research in to navigating from page to page whilst staying in immersive mode.
+# UA initiated immersive sessions
+
+## Motivation
+
+Current workflow to [request a WebXR session](https://www.w3.org/TR/webxr/#dom-xr-requestsession) requires user gesture activation. 2D Pages usually provide a button that users click or tap to trigger immersive mode.
+
+Some scenarios like in-vr navigation require entering immersive mode automatically without user gesture activation. This necessitates a standard mechanism for UAs to grant immersive sessions.
+
+## Design Goals
+
+The goal of this repository is to discuss and design a simple API for UAs to grant immersive sessions without an explicit user activation in specific scenarios.
+
+## Non Goals
+
+Define how a Virtual Reality or Augmented Reality browser would work [in alignment with the WebXR spec](https://github.com/immersive-web/webxr/blob/master/explainer.md#non-goals)
+
+## User Scenarios
+
+1. Immersive navigation, allowing UAs to implement seamless transitions between interlinked VR and AR content. Use cases include:
+
+    - Games that contains links to other games. Or links to friends' games.
+    - Personal blogs that link to affiliated VR sites or your friends blogs.
+    - Educational websites for a class, that link to the school's portal or other resources.
+    - User generated content linking to one another. Like Rec Room, VR Chat or Super Craft.
+    - Sponsored links. You have a VR world about photography and filmmaking that links to Photorama, B&H, GoPro's VR portals.
+    - Content portals in general (directories of games, immersive videos, collections).
+    - Personal storefronts that link to products hosted by eBay VR, Amazon VR. One could also easily incorporate product 3D models hosted externally in services the like of Sketchfab or Google Blocks.
+
+2. In an AR browser, virtual objects might be fetched from different URLs and rendered in physical context. Content is expected to always present at load time.
+
+3. Open in immersive mode. 2D windowed browsers might offer a way to open a link directly immersive mode, similar to "open in new tab" or "open in new page" actions offered today.
+
+4. Intents, deep linking, PWA, kiosk mode. Native aplications might request the browser to open content directly in immersive mode.
```

Comment:
Feels like these should be separated out, as they have very different implications.  Kiosk mode, in particular, feels like it's ripe for automatic session creation, but some deep linking and similar might still demand permissions.  It would be very good to summarize how these are handled for other permissions right now (e.g., camera and microphone access).

Reply:
Not it's broken down.